pom.xml

```diff
@@ -12,7 +12,7 @@
 
     <groupId>com.snowflake</groupId>
     <artifactId>snowflake-kafka-connector</artifactId>
-    <version>1.7.0</version>
+    <version>1.7.1</version>
```

Comment:
I don't see the JDBC version update anywhere, did I miss anything?

Reply:
I already pushed it to master.

Reply:
1ab1a1a9be86c8e429260344c8731dd56b793ee9